Content.Server/Explosion/Components/ExplosionLaunchedComponent.cs

```diff
@@ -19,7 +20,10 @@ void IExAct.OnExplosion(ExplosionEventArgs eventArgs)
 
             if (sourceLocation.Equals(targetLocation)) return;
 
-            var direction = (targetLocation.ToMapPos(Owner.EntityManager) - sourceLocation.ToMapPos(Owner.EntityManager)).Normalized;
+            var offset = (targetLocation.ToMapPos(Owner.EntityManager) - sourceLocation.ToMapPos(Owner.EntityManager));
+
+            //Don't normalize if the direction is center (0,0)
+            var direction = offset == Vector2.Zero ? offset : offset.Normalized;
```

Comment:
`Owner.TryThrow()` also tries to normalize the vector, which should probably also be fixed.

Also, if the vector is zero it should probably just not call `Owner.TryThrow()` in the first place

Reply:
I'll do the second one